server/issuers.go

```diff
@@ -42,7 +47,29 @@ func (c *Server) getLatestIssuer(issuerType string) (*Issuer, *handlers.AppError
 			Code:    500,
 		}
 	}
-	return &(*issuer)[0], nil
+	issuerByCohorts := breakCohorts(issuer)
+	lastIssuer := &(*issuerByCohorts)[issuerCohort]
+	if len(*lastIssuer) == 0 {
+		return nil, &handlers.AppError{
+			Message: "Cohort for this issuer not found",
+			Code:    404,
+		}
+	}
+	return &(*lastIssuer)[0], nil
+}
+
+// breaks a slice of []Issuer into a 2D slice with one slice per cohort.
+// This creates an empty slice with two slices, mainly [[],[]], one per cohort.
+// Then we iterate through all issuers, and position each issuer in the slice
+// corresponding to their cohort. Note: this only works for two cohorts. Maybe
+// we want to use a constant defining the number of cohorts?
```

Comment:
^^ makes sense to me, might as well do that now imo.

Reply:
finally decided to remove this and create a function `fetchIssuersByCohort`